### PR TITLE
feat: create React routes

### PR DESCRIPTION
**Pull request by ahn-nath**
Created React routes and base pages for each relevant view corresponding to a wireframe.

### CODE REVIEW

**Comment by tushar5526** [comment]
I see node_modules, is it intentional?

File: package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (tushar5526) [inline package.json]: I think package.json file in the root folder is of any use. Same goes for the node_modules

Comment (ahn-nath) [reply package.json]: No. I think we need the gitignore file with those. Let me change it.

Comment (tushar5526) [reply package.json]: I think, there is another package.json directly in the root. There is one that is in the `frontend` folder which is fine, but I can't see why we need `package.json` in the root folder, outside of the `frontend` folder.

**Comment by ahn-nath** [comment]
@tushar5526 
Resolved.

**Pull request merged**